vendor-extra/JatsContentBundle/src/EventListener/TitleListener.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\EventListener;
+
+use FluentDOM\DOM\Element;
+use Libero\ContentPageBundle\Event\CreateContentPageEvent;
+use function is_string;
+
+final class TitleListener
+{
+    private const FRONT_PATH = '/libero:item/jats:article/jats:front';
+    private const TITLE_PATH = self::FRONT_PATH.'/jats:article-meta/jats:title-group/jats:article-title';
```

Comment:
Had to split as the line is too long, and splitting over two lines doesn't see it indent how it probably should:

I'd expect

```php
    private const TITLE_PATH = '/libero:item/jats:article/jats:front/jats:article-meta/jats:title-group'
        .'/jats:article-title';
```

rather than

```php
    private const TITLE_PATH = '/libero:item/jats:article/jats:front/jats:article-meta/jats:title-group'
    .'/jats:article-title';
```

But probably better to avoid such long lines anyway?